ethcore/src/snapshot/tests/service.rs

```diff
@@ -104,6 +104,11 @@ fn restored_is_equivalent() {
 	}
 }
 
+
+// on windows the guards deletion (remove_dir_all)
+// is not happening (error directory is not empty).
+// So the test is disabled until windows api behave.
+#[cfg(not(target_os = "windows"))]
```

Comment:
Can you tell something more about the error returned? Why is it not happening? According to the [documentation](https://doc.rust-lang.org/std/fs/fn.remove_dir_all.html)

> Removes a directory at this path, after removing all its contents. Use carefully!

It should happen

Reply:
The error is a simple 'directory is not empty', looking at https://github.com/rust-lang/rust/issues/29497 which exists since 2015, I may have been a bit optimistic in saying that there will be a fix someday, we may probably want to use  https://crates.io/crates/remove_dir_all .

Reply:
Ok, I ran a test with the 'remove_dir_all' crate, and it is not better : we obtain an "os 32" error "the process cannot access this file because it is used by another process". Solving it will be probably a bit harder than expected, it should probably have its own issue and PR.